package.json

```diff
@@ -3,18 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.9.5",
```

Comment:
bhai bulma ke saath karle

Reply:
Arre uske UI components are not as good as material UI

Reply:
And bulma does not provide slider.